### ISSUE TITLE
Typos fix

### ISSUE THREAD

**Issue by iaso2h** [eI1on/pz-building-menu#15]
There are several duplicate `=` typos in the `Sandbox_**.txt`. 

https://github.com/eI1on/pz-building-menu/blob/93c556190c2a19072448b3310ab10de5908e9808/Contents/mods/BuildingMenu/media/lua/shared/Translate/EN/Sandbox_EN.txt#L320-L324

https://github.com/eI1on/pz-building-menu/blob/93c556190c2a19072448b3310ab10de5908e9808/Contents/mods/BuildingMenu/media/lua/shared/Translate/ES/Sandbox_ES.txt#L317-L321

https://github.com/eI1on/pz-building-menu/blob/93c556190c2a19072448b3310ab10de5908e9808/Contents/mods/BuildingMenu/media/lua/shared/Translate/KO/Sandbox_KO.txt#L315-L319

https://github.com/eI1on/pz-building-menu/blob/93c556190c2a19072448b3310ab10de5908e9808/Contents/mods/BuildingMenu/media/lua/shared/Translate/PTBR/Sandbox_PTBR.txt#L319-L323

https://github.com/eI1on/pz-building-menu/blob/93c556190c2a19072448b3310ab10de5908e9808/Contents/mods/BuildingMenu/media/lua/shared/Translate/RU/Sandbox_RU.txt#L319-L323

I've taken care of them in CN/CH version, but I think it's better for you @eI1on to deal with the rest, especially the ES version. Somehow with the [3cce680](https://github.com/eI1on/pz-building-menu/commit/3cce680f18e1c6f8ae73748bb528d162c0813915) commit being implemented I still got those errors from Git terminal client whenever I checkout a git branch.

```shell
error: failed to encode 'addons/Building_Menu_DaddyDirkie_Addon/Contents/mods/BuildingMenuDaddyDirkieAddon/media/lua/shared/Translate/ES/Tooltip_ES.txt' from UTF-8 to cp1252
```

**Comment by eI1on** [comment]
I hope they resolved after this commit [7e2a2df](https://github.com/eI1on/pz-building-menu/commit/7e2a2df0763b9cc9a8e8fdedf08fb3858fadb426)

**Comment by iaso2h** [comment]
I can confirm that the Git errors has been resolved.
